src/lib.rs

```diff
@@ -708,18 +613,114 @@ impl vfs::filesystem::FileSystem for FRFS {
 }
 
 /// Load a FRFS file.
-pub fn load<P: AsRef<path::Path> + Copy>(path: P) -> Result<FRFS> {
+pub fn load(path: impl AsRef<Path>) -> Result<FRFS> {
     FRFS::new(path)
 }
 
 /// Load a FRFS file from frfs::File.
-pub fn load_from_embed_file(file: File) -> Result<FRFS> {
-    FRFS::from_embed_file(file)
+pub fn load_from_file(file: File) -> Result<FRFS> {
+    FRFS::from_file(file)
```

Comment:
This implementation is absolutely wrong, because frfs can be nested, you want to read another frfs from frfs::File, you can't open the source directly, otherwise you will open a new instance of the current frfs